core/modules/path/path.admin.inc

```diff
@@ -123,14 +123,15 @@ function path_admin_edit($path = array()) {
  * @see path_admin_form_delete_submit()
  */
 function path_admin_form($form, &$form_state, $path = array('source' => '', 'alias' => '', 'langcode' => LANGUAGE_NONE, 'pid' => NULL)) {
+  $site_config = config('system.core');
   $form['source'] = array(
     '#type' => 'textfield',
     '#title' => t('Existing system path'),
     '#default_value' => $path['source'],
     '#maxlength' => 255,
     '#size' => 45,
     '#description' => t('Specify the existing path you wish to alias. For example: <code>node/28</code> or <code>taxonomy/term/1</code>.'),
-    '#field_prefix' => url(NULL, array('absolute' => TRUE)) . (config_get('system.core', 'clean_url') ? '' : '?q='),
+    '#field_prefix' => url(NULL, array('absolute' => TRUE)) . ($site_config->get('clean_url') ? '' : '?q='),
```

Comment:
Here, we could do something like:

```php
  $field_prefix = url(NULL, array('absolute' => TRUE)) . ($site_config->get('clean_url') ? '' : '?q=');
```

...and then reuse:

```php
  '#field_prefix' => $field_prefix,
```

It would save us 2 calls to `url()` 😉 ...but perhaps scope creep for this PR.

Reply:
Nice catch, Greg. How about if we leave this to a standalone PR, because if the review will be elevated to someone else, I don't want to confuse this PR with any code change other then just replacing `config_get`s with `site_config->get`s.

Reply:
But the other excellent catch of yours I have committed.